packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -68,24 +76,24 @@ export class CrossmintAASDK {
                 crossmintService: this.crossmintService,
             });
 
-            const address = await owner.getAddress();
-
-            const zDevProvider = await ZeroDevEthersProvider.init("ECDSA", {
-                projectId: getZeroDevProjectIdByBlockchain(chain),
-                owner,
-                opts: {
-                    providerConfig: {
-                        rpcUrl: getUrlProviderByBlockchain(chain),
-                    },
-                    paymasterConfig: {
-                        policy: "VERIFYING_PAYMASTER",
-                    },
+            const address = owner.address;
+
+            const ecdsaValidator = await signerToEcdsaValidator(publicClient, {
+                signer: owner,
+            });
+
+            const account = await createKernelAccount(publicClient, {
+                plugins: {
+                    sudo: ecdsaValidator,
                 },
             });
 
-            const evmAAWallet = new EVMAAWallet(zDevProvider, this.crossmintService, chain);
+            if (!isEVMBlockchain(chain)) {
+                throw new WalletSdkError(`The blockchain ${chain} is still not supported`);
+            }
```

Comment:
shouldn't we do this much earlier?

Reply:
actually I think we already have this code here: https://github.com/Crossmint/crossmint-sdk/pull/379/files#diff-44d9925b3eab949d3dc6cce03789d9160cfa1d1a34305326e05e39be67f70920R60-R62